website/docs/components/code-editor/partials/accessibility/accessibility.md

```diff
@@ -1,8 +1,10 @@
 ## Conformance rating
 
-<Doc::Badge @type="success">Conformant</Doc::Badge>
+<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>
 
-When used as recommended, there should not be any WCAG conformance issues with this component.
+When used without linting enabled, there should not be any WCAG conformance issues with this component.
+
+When linting is enabled, inline error indicator icons are not focusable. However, the errors are also displayed in a toggleable drawer beneath the code editor, providing an accessible alternative.
```

Comment:
```suggestion
When linting is enabled, the inline error indicator icons are not focusable. However, the errors are also displayed in a drawer beneath the code editor as an accessible alternative. The drawer is opened by pressing `cmd/ctrl + shift + m` while focused in the editor.
```

I think the build is failing because of `toggleable`. Just a suggestion to re-word this section.